docker/run-apache2.sh

```diff
@@ -48,11 +49,19 @@ a2ensite spinnaker
 
 cp docker/ports.conf.gen ports.conf
 
-sed -ie "s/{%DECK_HOST%}/$_DECK_HOST/g" ports.conf
-sed -ie "s/{%DECK_PORT%}/$_DECK_PORT/g" ports.conf
-
+sed -ie "s|{%DECK_HOST%}|$_DECK_HOST|g" ports.conf
+sed -ie "s|{%DECK_PORT%}|$_DECK_PORT|g" ports.conf 
 mv ports.conf /etc/apache2/ports.conf
 
+# Create a passphrase file to inject the SSL passphrase into apache's startup
+
+cp docker/passphrase.gen passphrase
+
+sed -ie "s|{%PASSPHRASE%}|$_PASSPHRASE|g" passphrase
```

Comment:
If it's just echoing a value passed via environment variable, why not just `exec:echo $PASSPHRASE` above? What's the value in writing the password to a file first?